main.py

```diff
@@ -372,7 +372,10 @@ def user_token(self, userid='core'):
     def check_docker(self):
         '''Check docker services status'''
         stack = 'docker-cloud stack inspect %s' % self.stackname
-        self._run(stack)
```

Comment:
@zelima Then I will have entire printed response, and my short version with `state`.